python/ray/_private/profiling.py

```diff
@@ -54,3 +77,137 @@ def profile(event_type, extra_data=None):
     if worker.mode == ray._private.worker.LOCAL_MODE:
         return NULL_LOG_SPAN
     return worker.core_worker.profile_event(event_type.encode("ascii"), extra_data)
+
+
+def chrome_tracing_dump(
```

Comment:
Most of code is from https://github.com/ray-project/ray/blob/1f3226ef866d1625c1d6d43ac553798dfbd941f5/python/ray/_private/state.py#L435

Reply:
should the naming reflect `perfetto`? 
Do they have the same output format?

Reply:
yes. Chrome tracing + perfetto requires the same output format https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview, but the original implementation only works at chrome tracing but not perfetto (we need metadata event to make it work at perfetto)